<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerDialog</class>
 <widget class="QDialog" name="ControllerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>292</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>756</width>
    <height>292</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>756</width>
    <height>292</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ПАРАМЕТРЫ МЦХ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="media.qrc">
    <normaloff>:/icons/window_icons/mainwindow_icon.png</normaloff>:/icons/window_icons/mainwindow_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">ControllerDialog{
	background: #f0f0f0;
}

QLineEdit{
	color: black;
    selection-background-color: #22cce7;
	border:  2 solid #b0b0b0;
}

QLineEdit:disabled{
	color: black;
	background: #f0f0f0;
}

QComboBox{
	color: black;
    selection-background-color: #22cce7;
	border:  2 solid #b0b0b0;
}

QComboBox:on{
	border: 2 solid #22cce7;
}

QComboBox::drop-down{
	background: transparent;
}

QComboBox::down-arrow{
	image: url(&quot;:/icons/styles/drop_down_arrow.png&quot;);
	width: 7;
	height: 7;
}

QComboBox::down-arrow:on {
	image: url(&quot;:/icons/styles/drop_down_arrow_up.png&quot;);
	width: 7;
	height: 7;
}

QComboBox QAbstractItemView QTextEdit {
	color: black;
    selection-background-color: #22cce7;
}

QComboBox QAbstractItemView QTextEdit:disabled{
	color: black;
	background: #f0f0f0;
}

QLabel:disabled{
    color: black;
}

QGroupBox {
    border: 0;
	background: transparent;
    margin-top: 20px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 3px;
}

QGroupBox::indicator {
	background: #f0f0f0;
    border: 2px solid black;  
    width: 15px;             
    height: 15px;           
}

QGroupBox::indicator::checked {
	 image: url(&quot;:/icons/styles/checkbox_arrow.png&quot;); 
}

QPushButton {
    border: 1px solid #c8c8c8;
    background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
                                stop:0 #e5e5e5, stop:1 white);
}

QPushButton:pressed{
	border: 2 solid #22cce7;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
                                stop:0 #e5e5e5, stop:1 white);
}

QMessageBox QPushButton {
	width: 100;
	height: 26;
    border: 1px solid #c8c8c8;
    background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
                                stop:0 #e5e5e5, stop:1 white);
}

QMessageBox QPushButton:pressed{
	width: 100;
	height: 26;
	border: 2 solid #22cce7;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
                                stop:0 #e5e5e5, stop:1 white);
}
</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="methodLabel">
         <property name="text">
          <string>Способ расчета:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="methodBox">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>26</height>
          </size>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>По массе</string>
          </property>
          <property name="icon">
           <iconset resource="media.qrc">
            <normaloff>:/icons/methodbox_icons/calc_1.png</normaloff>:/icons/methodbox_icons/calc_1.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По плотности</string>
          </property>
          <property name="icon">
           <iconset resource="media.qrc">
            <normaloff>:/icons/methodbox_icons/calc_2.png</normaloff>:/icons/methodbox_icons/calc_2.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По данным источника копирования</string>
          </property>
          <property name="icon">
           <iconset resource="media.qrc">
            <normaloff>:/icons/methodbox_icons/calc_3.png</normaloff>:/icons/methodbox_icons/calc_3.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="massFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox#massBox {
	border: 0;
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>358</width>
            <height>23</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="massLabel">
          <property name="text">
           <string>Масса, г</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="massEdit">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>26</height>
           </size>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="densityFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox#densityBox {
	border: 0;
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_27">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>36</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="densityLabel">
          <property name="text">
           <string>Плотность, г/мм3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="densityEdit">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>26</height>
           </size>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="coordBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Центр масс</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="xLabel">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="xEdit">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="yLabel">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yEdit">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="zLabel">
            <property name="text">
             <string>Z:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="zEdit">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="materialLabel">
         <property name="text">
          <string>Материал:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="materialEdit">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QLabel" name="hatchingLabel">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Штриховка</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="styleLabel">
         <property name="text">
          <string>Стиль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="styleBox">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>26</height>
          </size>
         </property>
         <property name="iconSize">
          <size>
           <width>90</width>
           <height>18</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Металл</string>
          </property>
          <property name="icon">
           <iconset resource="media.qrc">
            <normaloff>:/icons/stylebox_icons/hatch_1.png</normaloff>:/icons/stylebox_icons/hatch_1.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Естественный грунт</string>
          </property>
          <property name="icon">
           <iconset resource="media.qrc">
            <normaloff>:/icons/stylebox_icons/hatch_2.png</normaloff>:/icons/stylebox_icons/hatch_2.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Насыпной грунт</string>
          </property>
          <property name="icon">
           <iconset resource="media.qrc">
            <normaloff>:/icons/stylebox_icons/hatch_3.png</normaloff>:/icons/stylebox_icons/hatch_3.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Камень искусственный</string>
          </property>
          <property name="icon">
           <iconset resource="media.qrc">
            <normaloff>:/icons/stylebox_icons/hatch_4.png</normaloff>:/icons/stylebox_icons/hatch_4.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Железобетон</string>
          </property>
          <property name="icon">
           <iconset resource="media.qrc">
            <normaloff>:/icons/stylebox_icons/hatch_5.png</normaloff>:/icons/stylebox_icons/hatch_5.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Напряженный железобетон</string>
          </property>
          <property name="icon">
           <iconset resource="media.qrc">
            <normaloff>:/icons/stylebox_icons/hatch_6.png</normaloff>:/icons/stylebox_icons/hatch_6.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="angleLabel">
         <property name="text">
          <string>Угол:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="angleEdit">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2" rowspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>99</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>399</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="media.qrc"/>
 </resources>
 <connections/>
</ui>
